packages/nedit/SPECS/nedit.spec

```diff
@@ -77,6 +77,12 @@ desktop-file-install \
         --add-category "Development;" \
         %{SOURCE1}
 
+%pre
```

Comment:
I'm going to try to wrap `desktop-file-install` again and try to run this after it and somehow stage it in `$BUILDROOT`